### DISCUSSION TITLE
`@FocusState` support?

### DISCUSSION THREAD

**Discussion by joshuakcockrell** [skiptools/skip#142] [Q&A] [answered]
<img width="474" alt="Screenshot 2024-06-11 at 9 32 18 AM" src="https://github.com/skiptools/skip/assets/8979400/37dc55bf-0d80-4199-b1da-2559b45bd751">

Does Skip support any way to autofocus the user input on a `TextField`? Can that state be passed around as a `@Binding`?

**Comment by aabewhite** [comment, marked as answer]
We don't have FocusState support yet. We'll consider this a feature request! In the meantime I'd #if !SKIP it out of the Android build

**Comment by joshuakcockrell** [comment]
<img width="580" alt="Screenshot 2024-06-11 at 4 35 51 PM" src="https://github.com/skiptools/skip/assets/8979400/6b331974-a429-498a-b59b-7805e17c4f0f">

Is there a good pattern if I don't want to send this `FocusState` parameter to a `View` on Android? The compiler directives seem to totally break when used like this.

Maybe I just have two function calls?

**Reply by aabewhite** [reply]
Unfortunately you have to use separate function calls. I do want to add FocusState, so it'll be temporary. How many call sites do you have to duplicate for both forms of the function? Is it a ton?
